warehouse/email/__init__.py

```diff
@@ -186,6 +186,22 @@ def wrapper(request, user_or_users, **kwargs):
     return inner
 
 
+# Email templates for administrators.
+
+
+@_email("admin-new-organization-requested")
+def send_admin_new_organization_requested_email(
```

Comment:
We will likely want the recipient of these emails to be configurable, and a null value for it means "no email we'll go look at the admin queue ourself".

We may want to construct an email message directly https://github.com/pypa/warehouse/blob/439836ffd7570be4dec17be2146b543544fb11f0/warehouse/email/__init__.py#L165 and call https://github.com/pypa/warehouse/blob/439836ffd7570be4dec17be2146b543544fb11f0/warehouse/email/__init__.py#L58 directly.

Reply:
Currently we _only_ ever send emails to user accounts, but in this case it will likely be either a distribution list or a comma separated list of email addresses. Not all PyPI Admins should be notified unless they want to opt in.

@di, thoughts?

Reply:
Eh, I think just sending these to https://pypi.org/user/admin/ would be fine for now. Eventually when we support orgs/teams this could become more granular perhaps?

Reply:
that sparks a decent idea. a wrapper that sends to a pyramid principal rather than a user?

Reply:
> that sparks a decent idea. a wrapper that sends to a pyramid principal rather than a user?

@ewdurbin Are you thinking we should send the notification emails to every `"psf_staff"`, `"admins"`, or another Pyramid principal that we haven’t defined yet? I suppose this would be the same group with permission to approve/decline organizations in the admin interface?

Reply:
@ewdurbin If we want to send emails to all `"admins"` one approach would be to just grab all admin users from the database:

https://github.com/pypa/warehouse/blob/d38989f02ff7758e2178627ec5af788e53538b96/warehouse/accounts/services.py#L104-L106

https://github.com/pypa/warehouse/blob/d4611f07b47b6b684cabcb46fa5a2a86d3a43b57/warehouse/manage/views.py#L1021